modules/openapi-generator/src/test/resources/3_0/spring/issue_9789.yml

```diff
@@ -0,0 +1,46 @@
+openapi: 3.0.1
+info:
+  title: Test Issue #9789
+  version: v1
+paths:
+  /test:
+    get:
+      responses:
+        '200':
+          description: default response
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/ObjectWithNoRequiredParameter'
+components:
+  schemas:
+    ObjectWithNoRequiredParameter:
+      type: object
+      properties:
+        param1:
+          type: string
+    ObjectWithRequiredParameter:
+      type: object
+      properties:
+        param1:
+          type: string
+        param3:
+          type: string
+        param2:
+          type: string
```

Comment:
Can you make some of these nullable to excercise the nullability check as well?

Reply:
I added a nullable @ required parameter 🙂